src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -58,21 +60,53 @@ export const retrieveBeneficialOwners = async (req: Request, appData: Applicatio
 };
 
 export const retrieveManagingOfficers = async (req: Request, appData: ApplicationData) => {
+  const transactionId = appData.transaction_id;
+  const overseasEntityId = appData.overseas_entity_id;
   const companyOfficers = await getCompanyOfficers(req, appData[EntityNumberKey] as string);
-  if (companyOfficers) {
-    for (const officer of (companyOfficers.items || [])) {
-      logger.info("Loaded officer " + officer.officerRole);
-      if (officer.resignedOn === undefined) {
-        if (officer.officerRole === "managing-officer") {
-          const managingOfficer = mapToManagingOfficer(officer);
-          logger.info("Loaded Managing Officer " + managingOfficer.id + " is " + managingOfficer.first_name + ", " + managingOfficer.last_name);
-          appData.update?.review_managing_officers_individual?.push(managingOfficer);
-        } else if (officer.officerRole === "corporate-managing-officer") {
-          const managingOfficerCorporate = mapToManagingOfficerCorporate(officer);
-          logger.info("Loaded Corporate Managing Officer " + managingOfficerCorporate.id + " is " + managingOfficerCorporate.name);
-          appData.update?.review_managing_officers_corporate?.push(managingOfficerCorporate);
-        }
+  if (!companyOfficers) {return;}
```

Comment:
We could also check if `companyOfficers.items?.length === 0` too instead of checking this on line 81